Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.asymetricasteroids
+
+class Data {
+    // This is hard-coded <- BAD!
```

Comment:
I mean we have no choice lmao

Reply:
It's just default variables though no? I mean defining it on the PC and then trying to get that data is :^)